utils/payment-actions.js

```diff
@@ -1,4 +1,4 @@
-import { waitLoad } from '.'
+import { waitLoad } from './'
```

Comment:
Why did you change this file if you're not testing payment?

Reply:
I guess `prettier` wasn't properly setted before.